components/Cards/index.js

```diff
@@ -17,3 +17,91 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+const cardContainer = document.querySelector(".cards-container")
+// console.log(cardContainer)
+
+axios.get(`https://lambda-times-backend.herokuapp.com/articles`)
+.then(res =>{
```

Comment:
Another way we can get the same outcome without having to specify each of the article topics and iterating through them is by using a for...in loop which is checking for key values from your response like so...

```
axios
  .get('https://lambda-times-backend.herokuapp.com/articles')
  .then(response => {
    // articles variable to host the articles from response
    let articles = response.data.articles;
    // going through the keys of the articles object 
    for (topic in articles) {
       //  console.log(topic) you should get the articles "categories" === keys
      // . console.log(articles[topics]) you should get the array that that key(topic) is equal too and using that array we can iterate through it using a for each and appending our component from there
      articles[topic].forEach(article => {
        entry.appendChild(createCards(article));
      });
    }
  });
```